src/main/menu.ts

```diff
@@ -4,7 +4,7 @@ import isDev from "electron-is-dev";
 
 const editMenu: Electron.MenuItemConstructorOptions = {
   label: "Edit",
-  submenu: [{ role: "cut" }, { role: "copy" }, { role: "paste" }, { role: "selectall" }]
+  submenu: [{ role: "cut" }, { role: "copy" }, { role: "paste" }, { role: "selectall" }, { type: "separator" }, { label: "Toggle Comment", accelerator: "CommandOrControl+/" },],
```

Comment:
Can you remove this change if it doesn't work? After implementing ipc, I will implement it.

Reply:
fixed